src/core/Modal/Modal/Modal.tsx

```diff
@@ -137,6 +137,31 @@ class BaseModal extends Component<ModalProps> {
     }
   };
 
+  private getNodePrevState = (node: HTMLElement): null | PrevState => {
+    const ariaDisabledPrevState = node.getAttribute('aria-hidden');
```

Comment:
Why is this named ariaDisabledPrevState yet it containts aria-hidden attribute value? Why not make it ariaHiddenPrevState to be explicit?

Reply:
Should not be disabled. Due to original implementation where it covered both aria-hidden and role presentation. Will fix.

Reply:
Fixed